core/src/poh_recorder.rs

```diff
@@ -165,7 +168,7 @@ impl PohRecorder {
             .map(|slot| {
                 (
                     Some(slot * ticks_per_slot + grace_ticks),
-                    (slot + 1) * ticks_per_slot - 1,
+                    (slot + NUM_CONSECUTIVE_LEADER_SLOTS + 1) * ticks_per_slot - 1,
```

Comment:
I'd really prefer for this to be dynamic, i.e. poh recorder looks to see how many future slots belong to this node, instead of hard-coding it with constants.